client/src/components/Notification/NotificationItem.tsx

```diff
@@ -0,0 +1,48 @@
+import { Avatar, Grid, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+import { Notification } from '../../interface/Notification';
+import moment from 'moment';
+
+import NotificationsIcon from '@material-ui/icons/Notifications';
+
+interface Props {
+  notifications: Notification[];
+}
+
+export default function NotificationItem({ notifications }: Props): JSX.Element {
+  const classes = useStyles();
+  return (
+    <ul className={classes.listItemContainer}>
```

Comment:
Since we are using MUI, avoid using HTML tags where possible. 
For an unordered list you can use 'List' and 'ListItem' from MUI :)

Link: https://material-ui.com/components/lists/

Reply:
> Nice work, looks good!
> I made a couple of comments, please have a look.
> 
> Note: As notifications have to be created manually, please make a note to create a few notifications prior to our final presentation or you can delegate this task to one of the team-members.

I will be integrating notification with message and request, so we dont' have to manually create notifiactions